cordlib-contracts-states/src/main/kotlin/com/luxoft/blockchainlab/corda/hyperledger/indy/data/schema/CredentialDefinitionSchema.kt

```diff
@@ -26,7 +26,7 @@ object CredentialDefinitionSchemaV1 : MappedSchema(
         constructor(credentialDef: IndyCredentialDefinition) : this(
             credentialDef.credentialDefinitionId.toString(),
             credentialDef.schemaId.toString(),
-            credentialDef.credentialDefinitionId.getRevocationRegistryDefinitionId(IndyUser.REVOCATION_TAG).toString(),
+            credentialDef.credentialDefinitionId.possibleRevocationRegistryDefinitionId(IndyUser.REVOCATION_TAG).toString(),
```

Comment:
why do not you use verb here - "Get"?

Reply:
It somehow clashes with Kotlin's default getter. So in class `RevocationRegistryDefinitionId` I'm unable to add `ContainsCredentialDefinitionId` because it already consists of it.

Reply:
For some reason the clash just fade away...